submissions/AndriyM/hooli-style-popup/index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link rel="stylesheet" href="./styles.css" />
+        <title>Document</title>
+    </head>
+    <body>
+        <div class="wrapper">
+            <header class="header">
+                <div class="container">
+                    <nav class="nav">
+                        <ul class="nav__list">
+                            <li class="nav__item">
+                                <input class="popup__input" type="checkbox" id="check" />
+                                <label for="check" class="popup__label">
+                                    <img src="./img/pngwing.png" width="35" height="35" class="nav__img-link" alt="popup open button" />
+                                </label>
```

Comment:
Fix position of focused elements
![image](https://user-images.githubusercontent.com/16196199/187021165-ce2bedce-4fe8-48ba-a9ba-9494b14944b8.png)